specs/AppLifecycle/Restart/restartApi.markdown

```diff
@@ -0,0 +1,357 @@
+# Reunion Restart API Spec
```

Comment:
Please use [specs/spec_template.md](https://github.com/microsoft/WindowsAppSDK/blob/main/specs/spec_template.md)

You can find existing  specs @ [specs](https://github.com/microsoft/WindowsAppSDK/blob/main/specs) for reference examples. For example a really good simple example see [DeploymentAPI](https://github.com/microsoft/WindowsAppSDK/blob/main/specs/Deployment/DeploymentAPI.md). For more extensive specs [AppLifecycle Activation](https://github.com/microsoft/WindowsAppSDK/blob/main/specs/AppLifecycle/Activation/AppLifecycle%20Activation.md) and [Dynamic Dependencies]https://github.com/microsoft/WindowsAppSDK/blob/main/specs/dynamicdependencies/DynamicDependencies.md) are good examples.